packages/flutter_tools/lib/src/commands/build_ios.dart

```diff
@@ -277,7 +278,29 @@ class BuildIOSArchiveCommand extends _BuildIOSSubCommand {
     .toList();
   }
 
-  Future<void> _validateIconAssetsAfterArchive(StringBuffer messageBuffer) async {
+  ValidationResult? _createValidationResult(String title, List<ValidationMessage> messages) {
+    if (messages.isEmpty) {
+      return null;
+    }
+    final bool anyFailed = messages.any((ValidationMessage message) => message.type == ValidationMessageType.hint);
```

Comment:
do we need to account for `ValidationMessageType.error`?

Reply:
Oh i only use `hint`. let me add some comments.

Reply:
I added a comment, but just to be safe, i changed it to `=> message.type != ValidationMessage.information`.